src/project.cc

```diff
@@ -53,9 +53,6 @@ static std::vector<std::string> kBlacklistMulti = {
 // Blacklisted flags which are always removed from the command line.
 static std::vector<std::string> kBlacklist = {
     "-c", "-MP", "-MD", "-MMD", "--fcolor-diagnostics",
-
-    // This strips path-like args but is a bit hacky.
-    "/", "..",
```

Comment:
Do tests still pass with this off? Sync and run `./waf ; ./build/app --test-unit`

Reply:
It passes.